### DISCUSSION TITLE
running server on ubuntu arm64

### DISCUSSION THREAD

**Discussion by darkjezza** [AscensionGameDev/Intersect-Engine#1820] [Q&A] [answered]
hello I'm having trouble running the server on Ubuntu arm64, i have installed mono 6.12.0.182 iv tried running LD_LIBRARY_PATH=. mono "Intersect Server.exe" I'm not sure where to go from here
Canonical-Ubuntu-20.04-aarch64
mono version 6.12.0.182

Mono JIT compiler version 6.12.0.182 (tarball Tue Jun 14 22:41:13 UTC 2022)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
        TLS:           __thread
        SIGSEGV:       normal
        Notifications: epoll
        Architecture:  arm64
        Disabled:      none
        Misc:          softdebug
        Interpreter:   yes
        LLVM:          yes(610)
        Suspend:       preemptive
        GC:            sgen (concurrent by default)

![de99a13487677da90c41061a6c2585f2(1)](https://github.com/AscensionGameDev/Intersect-Engine/assets/80877059/7fe8098f-c25e-457c-abb9-52605b6b84dd)

**Comment by Arufonsu** [comment, marked as answer]
dupe of #799
